blueprints/comments.py

```diff
@@ -0,0 +1,34 @@
+from flask import Blueprint
+from bson import ObjectId
+
+from .utils import mongo_connect
+
+comment = Blueprint('comment', __name__)
+
+
+@comment.route('/add_comment/', methods=["POST"])
+def add_comment():
+    """
+    AJAX route to take a comment as JSON and add it to the database
+    """
+    data = request.data
+    commentsDict = json.loads(data)
+    if "parent" not in commentsDict:
+        commentsDict["parent"] = ""
+    else:
+        commentsDict["parent"] = ObjectId(
+            commentsDict["parent"])
```

Comment:
If the opening parenthesis is on a different line from the content, then the closing parenthesis should be on the next line, otherwise, have the class and all content on one line. 
```ObjectId(commentsDict["parent"])``` OR
```
ObjectId(
     commentsDict["parent"]
)
```